libs/net/dal/Migrations/1.0.52/Up/PostUp/00-ChartTemplate.sql

```diff
@@ -22,7 +22,7 @@ INSERT INTO public."chart_template" (
 @using System.Linq
 @using TNO.Entities
 @{
-  var group = Settings.GroupBy switch
+  var group = SectionSettings.GroupBy switch
```

Comment:
This won't fix it for DBs that already have applied the migration, but we don't have any supported charts yet, so it can be resolved manually.